web/src/settings_banner.ts

```diff
@@ -0,0 +1,136 @@
+import $ from "jquery";
+
+import * as banners from "./banners.ts";
+import {$t} from "./i18n.ts";
+import * as settings_data from "./settings_data.ts";
+
+const UPGRADE_ACCESS_BANNER: banners.Banner = {
```

Comment:
You can do an additional `import type {Banner} from "./banners.ts";` and avoid having to write `banners.Banner` again and again.